### DISCUSSION TITLE
How can I change the font size of a label?

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#9397] [Help] [answered]
Hi everyone?
How can i change the font-size of a label.
It doesn't work with custom css.
@melohagan

**Comment by andz-bb** [comment, marked as answer]
hey @rafikApp 

are you talking about changing the font-size of form field labels? You could write custom styling inside an embed component for that field label class. I used this in an embed component and it was able to apply the styles I wanted:

```
<style>
  .spectrum-FieldLabel {
    font-size: 16px;
    color: cadetblue;
  }
</style>
```

![Screenshot 2023-01-20 at 11 37 07](https://user-images.githubusercontent.com/110921612/213686347-3ccc7399-1ab5-4562-aab8-b0440afa7c82.png)

Hope this helps!

**Reply by rafikApp** [reply]
Hi @andz-bb Thank you for the help.
I want to modify a single label, not all the labels

**Reply by andz-bb** [reply]
hey @rafikApp 

For selecting a single label you'll need to use your browser's developer tools (F12 on chrome) to inspect the element you want to style in order to find that component's ID. Each Budibase component has a unique ID that is appended to that element's class so we can use that to select our specific label and style it.

![image](https://user-images.githubusercontent.com/110921612/213697328-7c84543e-e825-4bca-b73d-c1389f227703.png)

In my case I have changed the size and colour of the Notes label. To do this, I took the ID from the parent div and then used the `spectrum-FieldLabel` class to select the label div within that parent specifically.

Here's the CSS I used in my embed component to target that one label:

```
<style>
  .c03ed9c13c6194cb5894ca3665d60fce0-dom .spectrum-FieldLabel {
    font-size: 16px;
    color: dodgerblue;
}
</style>
```

Hope I explained it OK, it's a bit awkward if you're not familiar with browser developer tools so if you need any more help let me know!
